target/linux/ath79/image/lzma-loader/src/Makefile

```diff
@@ -16,7 +16,7 @@
 #
 
 LOADADDR	:=
-LZMA_TEXT_START	:= 0x80a00000
+LZMA_TEXT_START	:= 0x81800000
```

Comment:
So, there is another LZMA_TEXT_START here, which is provided as argument for the linker.

We really need to find somebody that understand this stuff properly.